ui/src/routes/organizations/$orgId/products/$productId/repositories/$repoId/-types.ts

```diff
@@ -146,16 +146,16 @@ export type PackageManager = (typeof packageManagers)[number]['id'];
 
 export const reportFormats = [
   {
-    id: 'CycloneDx',
-    label: 'CycloneDX SBOM',
+    id: 'CycloneDX',
+    label: 'CycloneDX SBOM (both JSON and XML)',
   },
   {
     id: 'SpdxDocument',
-    label: 'SPDX Document',
+    label: 'SPDX Document (both JSON and YAML)',
   },
   {
     id: 'PlainTextTemplate',
-    label: 'NOTICE file',
+    label: 'NOTICE file (both NOTICE_DEFAULT and NOTICE_SUMMARY)',
```

Comment:
Here, the "NOTICE_" prefixes could be dropped for shortness, and say "(DEFAULT and SUMMARY formats)".